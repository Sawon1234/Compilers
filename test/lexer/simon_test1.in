interface Integer<Integer> extends Integer<Integer> {
    
}