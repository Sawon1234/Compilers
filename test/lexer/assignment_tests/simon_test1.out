interface Name < Name > extends Name < Name > { }